common/religion/doctrines/00_core_tenets.txt

```diff
@@ -1907,127 +2002,205 @@
 	#############################
 	# Syncretic Faith Tenets	#
 	#############################
```

Comment:
I suggest moving them  to a separate file

Reply:
See https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/21/files#r518731788.